### PR TITLE
ROX-20293: Create GHA workflow to install the addon and enable it for integration

### PR DESCRIPTION
**Pull request by kovayur**
## Description
Add the new GHA workflow that installs the addon and enable it for integration.

## Checklist (Definition of Done)

- [ ] Unit and integration tests added
- [ ] Added test description under `Test manual`
- [ ] Documentation added if necessary (i.e. changes to dev setup, test execution, ...)
- [ ] CI and all relevant tests are passing
- [ ] Add the ticket number to the PR title if available, i.e. `ROX-12345: ...`
- [ ] Discussed security and business related topics privately. Will move any security and business related topics that arise to private communication channel.
- [ ] Add secret to app-interface Vault or Secrets Manager if necessary
- [ ] RDS changes were e2e tested [manually](../docs/development/howto-e2e-test-rds.md)
- [ ] Check AWS limits are reasonable for changes provisioning new resources

## Test manual

**TODO:** Add manual testing efforts

```
# To run tests locally run:
make db/teardown db/setup db/migrate
make ocm/setup OCM_OFFLINE_TOKEN=<ocm-offline-token> OCM_ENV=development
make verify lint binary test test/integration
```

### CODE REVIEW

File: .github/actions/setup-ocm-cli/action.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - name: Ensure OCM CLI is installed
+      run: |
+        INSTALL_DIR=${HOME}/.local/bin
+        if [[ -f ${INSTALL_DIR}/ocm ]]; then
+          echo "OCM already installed in ${INSTALL_DIR}/ocm"
+        else
+          echo "Installing ocm in ${INSTALL_DIR}/ocm"
+          mkdir -p ${INSTALL_DIR}
+          DOWNLOAD_URL="https://github.com/openshift-online/ocm-cli/releases/download/${{ inputs.version }}/ocm-linux-amd64"
+          curl -sL "${DOWNLOAD_URL}" > ${INSTALL_DIR}/ocm
```
Comment (porridge) [inline .github/actions/setup-ocm-cli/action.yaml:30]:
```suggestion
          curl --fail --retry 5 -sL "${DOWNLOAD_URL}" > ${INSTALL_DIR}/ocm
```

File: .github/actions/setup-ocm-cli/action.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          chmod +x ${INSTALL_DIR}/ocm
+        fi
+
+        echo "$HOME/.local/bin" >> $GITHUB_PATH
```
Comment (porridge) [inline .github/actions/setup-ocm-cli/action.yaml:34]:
```suggestion
        echo "${INSTALL_DIR}" >> $GITHUB_PATH
```

**Review by porridge — APPROVED** [review]
LGTM overall, but please apply the inline nitpicks

**Comment by ebensh** [comment]
@kovayur Can we close this out now?

**Pull request closed without merging**